include/boost/geometry/util/numeric_cast.hpp

```diff
@@ -30,21 +29,11 @@ struct numeric_caster
     }
 };
 
-// Specialization for Boost.Rational
-template <typename Target, typename T>
-struct numeric_caster<Target, rational<T>>
-{
-   static inline Target apply(rational<T> const& source)
-    {
-        return boost::rational_cast<Target>(source);
-    }
-};
```

Comment:
This was placed here in last commit, but it is more consistent with other approach if it is all in `rational.hpp` (it was there before as well)